packages/ui/components/apps/appCard.test.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable playwright/missing-playwright-await */
+import { render, screen } from "@testing-library/react";
+
+import { AppCard } from "./AppCard";
+
+describe("Tests for AppCard component", () => {
+  const mockApp = {
+    logo: "/path/to/logo.png",
+    name: "Test App",
+    slug: "test-app",
+    description: "Test description for the app.",
+    categories: ["calendar"],
+    concurrentMeetings: true,
+    teamsPlanRequired: false,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  } as any;
```

Comment:
why are you using any here ? can't you use type 'App' that is required in AppCardProps?

Reply:
We made some changes to remove this any 😄